include/great_expectations/great_expectations.yml

```diff
@@ -124,6 +124,6 @@ data_docs_sites:
       class_name: DefaultSiteIndexBuilder
 
 anonymous_usage_statistics:
-  data_context_id: 28fc8bae-d91c-4ca3-8c9f-b0270ea34e7d
-  enabled: true
+  data_context_id: 00000000-0000-0000-0000-000000000000
+  enabled: false
```

Comment:
Hi @jeffkpayne - could we change this `data_context_id` to the one that I added in the comment on `.ge_store_backend_id`? Or something else that looks artificial, but is also not all 0s (we use the all 0s for our testing). Would you mind also enabling usage stats? I talked to our analytics folks, and we figured it might be useful to see how often people were using this tutorial. If you have serious concerns about this, I'm open to leaving this as it is.

Reply:
Ah, yes! Lol. Sorry, I guess I picked the same testing value that you did. I'll take that as a compliment! Thanks. Doing that now.